code/datums/trading/food.dm

```diff
@@ -72,6 +86,7 @@
 		"hail_deny"          = "We do not take orders from rude customers.",
 		"trade_complete"     = "Thank you, sir, for your patronage.",
 		"trade_blacklist"    = "No, that is very odd. Why would you trade that away?",
+		"what_want"          = "It's very odd, but I need",
```

Comment:
```suggestion
		"what_want"          = "It's very odd, but I need.",
```

Reply:
- unresolved - just now looked at the code again and thought about it - this conversation option lists a pool of items after it to tell the merchant what the trader wants right now. 
" It's very odd, but I need. kitchen knife, plastic fork." is very derp.

Re-committing previous version and resolving this and it's sister thread with same. [Code is like this on all traders that buy items.]